### DISCUSSION TITLE
Import chatgpt chat histories

### DISCUSSION THREAD

**Discussion by antonrv** [mckaywrigley/chatbot-ui#1506] [Ideas]
It would be nice to have a way to import chatgpt chat histories into supabase via chatbot UI. That would allow OpenAI ChatGPT users to seamlessly migrate from a monthly subscription model to a pay-per-use API-based model, more cost-effective for most of users I bet.

**Comment by secondtruth** [comment]
@sebiweise is currently working on this feature: #1274
